packages/rum-core/src/performance-monitoring/capture-hard-navigation.js

```diff
@@ -77,6 +77,29 @@ function shouldCreateSpan(start, end, baseTime, transactionEnd) {
   )
 }
 
+/**
+ * Both Navigation and Resource timing level 2 exposes these below information
+ *
+ * for CORS requests without Timing-Allow-Origin header, transferSize & encodedBodySize will be 0
```

Comment:
Maybe we should add this to the docs.

Reply:
Where we can add this? I wasn't sure which section would fit this well.

Reply:
Added a issue to track to do in separate PR #320